plugins/dev9ghzdrk/pcap_io.cpp

```diff
@@ -261,11 +280,14 @@ int pcap_io_get_dev_num()
 
 	pcap_freealldevs(alldevs);
 
+
```

Comment:
I'd remove the extra lines, you can have empty one before return, same for the others.